### PR TITLE
fix social sharing modal fixes #2496

### LINKED ISSUE #2496: Lantern social sharing page not looking good on IE10
image

### PR DESCRIPTION
**Pull request by fffw**
Fixes https://github.com/getlantern/lantern/issues/2496

Adjusted width and layout of the modal.

The scroll bar is shown during when twitter script is inserting iframe. It will disappear once the content is inserted, but can't be eliminated by using css 'overflow: hidden'.

I'll try using fixed width iframe element directly to see if it works.

### CODE REVIEW

**Comment by fffw** [comment]
Ok, now it works fine on Chrome, Firefox and IE 11. @myleshorton Would you like to take a look?

File: src/github.com/getlantern/lantern-ui/sass/_modals.sass [hunk 1]
```diff
@@ -17,6 +17,26 @@ body[dir=rtl] .modal-footer
   +single-transition(opacity, 0.3s, ease)
   top: 10% // same as .modal.fade.in
 
+#welcomescreen
```
Comment (fffw) [inline src/github.com/getlantern/lantern-ui/sass/_modals.sass]: The dirty ways to make the modal look good

**Comment by myleshorton** [comment]
Thanks @fffw! I'll look at this today.

**Comment by myleshorton** [comment]
This is looking pretty good to me on IE 10, but the Facebook share button seems to take forever to load. It also doesn't seem to load at all for me on FF, and I get this message in both:

"Invalid App Id: Must be a number or numeric string representing the application id"

**Comment by fffw** [comment]
Oh I forgot to mention. The Facebook share button did take a long time to load or not load at all sometimes. It took 1.8 min to load during my last test.

The warning doesn't prevent the button from being shown, but maybe an App Id can accelerate it? Let me have a try.

**Comment by fffw** [comment]
Facebook button opens super fast once we supply an App Id.

Let me checking https://github.com/getlantern/lantern/issues/2509 and try to resolve it in same PR.

**Comment by myleshorton** [comment]
Sweet @fffw!

**Comment by fffw** [comment]
https://github.com/getlantern/lantern/issues/2509 seems fixed by this PR.

**Comment by myleshorton** [comment]
Sounds great @fffw. The Facebook share button is still loading super slow for me, but we can open another ticket for that. Seems odd that would happen.

**Comment by myleshorton** [comment]
Looking good on all platforms. Merging.

**Pull request merged**